pycbc/waveform/ringdown.py

```diff
@@ -65,11 +64,47 @@ def props(obj, required, **kwargs):
 
     return input_params
 
+def format_lmns(lmns):
+    """ Checks if the format of the parameter lmns is correct, returning the
+    appropriate format if not, and raise an error if nmodes=0
+    """
+    # Case 0: the lmns is in the right format
+    if len(lmns[0]) == 3:
+        lmns = lmns
+
+    else:
+        # Case 1: the list is in a string with or without comma,
+        # "[221, 331]" or "[221 331]"
+        if lmns[0] == '[' and lmns[-1] == ']':
+            lmns = lmns.strip('[]')
+
+        # Case 2: a list with only one string with or without comma,
+        # ["221' '331"] or ["221', '331"]
+        elif len(lmns[0]) > 3:
+            lmns = lmns[0]
```

Comment:
Because in this case there is only one item in the list, which is a string. This is what the inference workflow configparser was returning. So `lmns[0] = "221' '331"` or `lmns[0] = "221', '331"` in the example